tests/compute/subresources/pausing.go

```diff
@@ -67,7 +68,7 @@ var _ = compute.SIGDescribe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com
 			vmi = libvmops.RunVMIAndExpectLaunch(libvmifact.NewCirros(), timeout)
 		})
 
-		It("[test_id:4597]should signal paused state with condition", func() {
+		It("[test_id:4597]should signal paused state with condition", decorators.Conformance, func() {
```

Comment:
Again I hate the `BeforeEach`, it doesn't bring value. Quit contrary I need to scroll up and back down